src/legacy/core_plugins/kibana/public/dashboard/np_ready/dashboard_app_controller.tsx

```diff
@@ -660,7 +669,8 @@ export class DashboardAppController {
           // temporary solution is to delay $location updates to next digest cycle
```

Comment:
Just noticed this comment - is the eval async still necessary now that the complete state management is migrated?

Reply:
Removed, hope ci won't argue :D